### ISSUE TITLE
Individually included files are not seen

### ISSUE THREAD

**Issue by lawzava** [serverless/serverless#5608]
# This is a Bug Report

## Description

Error: "No file matches include / exclude patterns"

Config:
```
...

package:
 exclude:
   - ../**
 individually: true

functions:
  authorizerFunc:
    handler: bin/authorizer
    package:
      include:
        - ../bin/authorizer
```

The binary exists. Compiled as `env GOOS=linux go build -ldflags="-s -w" -o bin/authorizer authorizer/*.go`

It happens only with the `1.35.0` version. When downgraded to `1.34.1` it works as expected.

## Additional Data

Stack Trace:
```
ServerlessError: No file matches include / exclude patterns
    at globby.then.allFilePaths (/usr/lib/node_modules/serverless/lib/plugins/package/lib/packageService.js:209:13)
From previous event:
    at Package.resolveFilePathsFunction (/usr/lib/node_modules/serverless/lib/plugins/package/lib/packageService.js:159:48)
    at Package.packageFunction (/usr/lib/node_modules/serverless/lib/plugins/package/lib/packageService.js:119:17)
    at _.map.functionName (/usr/lib/node_modules/serverless/lib/plugins/package/lib/packageService.js:58:21)
    at arrayMap (/usr/lib/node_modules/serverless/node_modules/lodash/lodash.js:639:23)
    at Function.map (/usr/lib/node_modules/serverless/node_modules/lodash/lodash.js:9556:14)
    at Package.packageService (/usr/lib/node_modules/serverless/lib/plugins/package/lib/packageService.js:46:29)
From previous event:
    at Object.package:createDeploymentArtifacts [as hook] (/usr/lib/node_modules/serverless/lib/plugins/package/package.js:64:10)
    at BbPromise.reduce (/usr/lib/node_modules/serverless/lib/classes/PluginManager.js:391:55)
From previous event:
    at PluginManager.invoke (/usr/lib/node_modules/serverless/lib/classes/PluginManager.js:391:22)
    at PluginManager.spawn (/usr/lib/node_modules/serverless/lib/classes/PluginManager.js:409:17)
    at Deploy.BbPromise.bind.then (/usr/lib/node_modules/serverless/lib/plugins/deploy/deploy.js:117:50)
From previous event:
    at Object.before:deploy:deploy [as hook] (/usr/lib/node_modules/serverless/lib/plugins/deploy/deploy.js:107:10)
    at BbPromise.reduce (/usr/lib/node_modules/serverless/lib/classes/PluginManager.js:391:55)
From previous event:
    at PluginManager.invoke (/usr/lib/node_modules/serverless/lib/classes/PluginManager.js:391:22)
    at PluginManager.run (/usr/lib/node_modules/serverless/lib/classes/PluginManager.js:422:17)
    at variables.populateService.then.then (/usr/lib/node_modules/serverless/lib/Serverless.js:157:33)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
From previous event:
    at Serverless.run (/usr/lib/node_modules/serverless/lib/Serverless.js:144:8)
    at serverless.init.then (/usr/lib/node_modules/serverless/bin/serverless:44:28)
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information -----------------------------
     OS:                     linux
     Node Version:           6.15.1
     Serverless Version:     1.35.0
 
Exited with code 1
```

**Comment by MacMcIrish** [comment]
Likely fixed by https://github.com/serverless/serverless/pull/5602

**Comment by dschep** [comment]
Yup. Please upgrade to v1.35.1 and let us know if your issue is resolved

**Comment by pmuens** [comment]
Closing since it should've been fixed in `v1.35.1`. Feel free to re-open if this is still an issue...
